gateway/gateway.go

```diff
@@ -114,8 +114,13 @@ var BatchEvent = []byte(`
 const (
 	DELIMITER                 = string("<<>>")
 	eventStreamSourceCategory = "eventStream"
+	extractEvent              = "extract"
 )
 
+// EventTypesSkipUserCheck is a list of event types that
+// do not require an user id or anonymous id
+var EventTypesSkipUserCheck = []string{extractEvent}
```

Comment:
There is no need to export this variable, we should make it unexported.

Reply:
Addressed!